src/components/EditableInput/EditableInput.tsx

```diff
@@ -47,17 +47,17 @@ export const EditableNodeItem = ({ name, targetItemId, onEditableSave, children
     };
 
     return (
-        <div>
+        <div data-test-id="editable-node-container">
             {showInput ? (
                 <div className="tw-flex tw-items-center">
                     <div
                         data-test-id="node-editable"
```

Comment:
This one stayed a `node-editable` and the new one is `editable-node-container`. For consistency could we rename them to `editable-input`?